src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DeleteDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,180 @@
+                        "type": "urn:ed-fi:api:data-conflict:dependent-item-exists",
+                        "title": "Dependent Item Exists",
+                        "status": 409,
+                        "correlationId": null,
+                        "validationErrors": {},
+                        "errors": []
+                    }
+                  """
+
+        @ignore
+        Scenario: 08 Verify response when deleting a student with more than one reference
+            Given the system has these "schools"
+                  | schoolId | nameOfInstitution | gradeLevels                                                                      | educationOrganizationCategories                                                                                        |
+                  | 4005     | Test school       | [ {"gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#First Grade"} ] | [ {"educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#school"} ] |
+              And the system has these "contacts"
+                  | contactUniqueId | firstName          | lastSurname       |
+                  | "123"           | "contactFirstname" | "contactLastname" |
+              And the system has these "students" references
+                  | studentUniqueId | birthDate  | firstName   | lastSurname |
+                  | "987"           | 2017-08-23 | "firstname" | "lastname"  |
+              And the system has these "studentSchoolAssociations"
+                  | entryDate  | schoolReference    | studentReference           | entryGradeLevelDescriptor                        |
+                  | 2021-07-23 | {"schoolId": 4005} | {"studentUniqueId": "987"} | uri://ed-fi.org/GradeLevelDescriptor#First grade |
+              And the system has these "studentContactAssociations"
+                  | contactReference           | studentReference           |
+                  | {"contactUniqueId": "123"} | {"studentUniqueId": "987"} |
+             When a DELETE request is made to referenced resource "/ed-fi/students/{id}"
+             Then it should respond with 409
+              And the response body is
+                  """
+                    {
+                        "detail": "The requested action cannot be performed because this item is referenced by existing StudentContactAssociation, StudentSchoolAssociation item(s).",
+                        "type": "urn:ed-fi:api:data-conflict:dependent-item-exists",
+                        "title": "Dependent Item Exists",
+                        "status": 409,
+                        "correlationId": null,
+                        "validationErrors": {},
+                        "errors": []
+                    }
+                  """
+
+        @ignore
+        Scenario: 09 Ensure clients can delete a descriptor that is not beign used by any resource
```

Comment:
This duplicates other scenarios above where a successful delete occurs.

What we're missing is a scenario covering an attempt to delete a Descriptor that is in use. For example, you could adapt Scenario 6. The `Given` section only needs the `School` setup. Then try to delete that's school's `GradeLevelDescriptor`. Should result in a 409 conflict.